packages/backend/src/main/resources/changesets/fixtures-1.1.sql

```diff
@@ -0,0 +1,53 @@
+INSERT INTO public.room (id, name)
```

Comment:
Please add context to the fixtures..sql.
To ensure that if somebody change the changelog config that this fixture will be correct used.
```suggestion
--changeset fixtures:1.1 contextFilter:fixtures
INSERT INTO public.room (id, name)
```